### PR TITLE
[DSI-7053] Show and hide approvers in organisations table

### PR DESCRIPTION
**Pull request by ipredoi**
## Summary
**Jira ticket:** https://dfe-secureaccess.atlassian.net/browse/DSI-7053
## Changes
- Enclosed the approvers list within a details component in the user's organisations table

## Related PRs
https://github.com/DFE-Digital/login.dfe.ui-toolkit/pull/130

### CODE REVIEW

**Review by shravani-kondaparti — APPROVED** [review]
verified the changes in test env

File: src/app/organisations/views/organisations.ejs [hunk 1]
```diff
@@ -151,30 +151,30 @@
...
                                                         <% for (let a = 0; a < organisation.approvers.length; a++) { %>
-                                                            <li><%= organisation.approvers[a].email %></li>
+                                                            <li>
+                                                                <%= organisation.approvers[a].email %>
+                                                            </li>
                                                         <% } %>
-                                                    </ul>
-                                                </dl>
-                                            </div>
+                                                    </ul>
```
Comment (insacuri) [inline src/app/organisations/views/organisations.ejs:167]: Can you remove the random extra spaces in this line?

**Review by insacuri — DISMISSED** [review]
Good to go once my comment is addressed.

File: src/app/organisations/views/organisations.ejs (hunk 1 above)
Comment (ipredoi) [reply src/app/organisations/views/organisations.ejs]: @insacuri the white spaces were removed

**Review by shravani-kondaparti — APPROVED** [review]

**Review by insacuri — APPROVED** [review]

**Pull request merged**